common/devdoc/interlocked_hl_requirements.md

```diff
@@ -0,0 +1,193 @@
+# interlocked_hl requirements
+=============
```

Comment:
while you are moving these 4 .MD files, can we remove the "====" part and make sure the header has the "\#" like in this file?

So all the requirements.md files are

```
# name requirements

## Overview
...
```

Reply:
modified